solr/contrib/ltr/README.md

```diff
@@ -20,38 +20,17 @@ the techproducts example please follow these steps.
     `ant dist`
     `ant server`
 
-2. Run the example to setup the index
+2. Run the example to setup the index, enabling the ltr plugin
 
-   `./bin/solr -e techproducts`
+   `./bin/solr -e techproducts -Dsolr.ltr.enabled=true`
 
-3. Stop solr and install the plugin:
-     1. Stop solr
-
-        `./bin/solr stop`
-     2. Create the lib folder
-
-        `mkdir example/techproducts/solr/techproducts/lib`
-     3. Install the plugin in the lib folder
-
-        `cp build/contrib/ltr/solr-ltr-7.0.0-SNAPSHOT.jar example/techproducts/solr/techproducts/lib/`
-     4. Replace the original solrconfig with one importing all the ltr components
-
-        `cp contrib/ltr/example/solrconfig.xml example/techproducts/solr/techproducts/conf/`
-
-4. Run the example again
-
-   `./bin/solr -e techproducts`
-
-   Note you could also have just restarted your collection using the admin page.
-   You can find more detailed instructions [here](https://wiki.apache.org/solr/SolrPlugins).
-
-5. Deploy features and a model
+3. Deploy features and a model
```

Comment:
Yes, could temporarily mention here if you like, but don't have to. I've started on the Solr Ref Guide documentation at https://cwiki.apache.org/confluence/display/solr/Result+Reranking - very much work-in-progress including title of that page.

For the github log, the plan is for the README in the end to not duplicate Solr Ref Guide or javadocs content but to provide a pointer into the documentation.